notebooks/community/pipelines/google_cloud_pipeline_template.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
<p>Curious, all three of these custom components could be done with prebuilt GCPC components.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>